### ISSUE TITLE
Table creation causing error

### ISSUE THREAD

**Issue by AlexandreWonderwalls** [cancan101/airtable-db-api#212]
I try to create table from this interface but it cause an error. 

I have this code : 
key = "keyXXXX"
engine = create_engine('airtable://:'+key)
factory = sessionmaker(bind=engine)
session = factory()
ModelBase.metadata.create_all(engine)

## Error :
  response.raise_for_status()
  File "\venv\lib\site-packages\requests\models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: ('404 Client Error: Not Found for url: https://api.airtable.com/v0/appXXXX/VentilationProcessed?pageSize=1&maxRecords=1', "404 Client Error: Not Found for url: https://api.airtable.com/v0/appXXXX/VentilationProcessed?pageSize=1&maxRecords=1 [Error: {'type': 'TABLE_NOT_FOUND', 'message': 'Could not find table VentilationProcessed in application appXXXX'}]")

Do I misunderstood something? Or this is an issue? And, is their an alternative?

**Comment by cancan101** [comment]
What are you trying to accomplish with that code? Are you trying to create a Table within a given Airtable base? Table creation is not supported by this library. 

If you are just trying to query an existing table, make sure the table name is spelled correctly, etc. That error looks to be coming from Airtable itself saying the table `VentilationProcessed` does not exist in the base `appXXXX`.

**Comment by AlexandreWonderwalls** [comment]
Yes I'm trying to create a Table. I will use another thing to create the table,  I think.
